bindep.txt

```diff
@@ -1,9 +1,6 @@
-openssl-devel [platform:rpm compile]
-gcc [platform:rpm compile]
-libcurl-devel [platform:rpm compile]
-libxml2-devel [platform:rpm compile]
-python3-pycurl [platform:rpm]
-python3-netaddr [platform:rpm]
-python3-jmespath [platform:rpm]
-python3-passlib [platform:rpm epel]
-qemu-img [platform:rpm]
+gcc [compile platform:centos-8 platform:rhel-8]
+libcurl-devel [compile platform:centos-8 platform:rhel-8]
+libxml2-devel [compile platform:centos-8 platform:rhel-8]
+openssl-devel [compile platform:centos-8 platform:rhel-8]
+python38-devel [compile platform:centos-8 platform:rhel-8]
+qemu-img [platform:centos-8 platform:rhel-8]
```

Comment:
Is this needed on the controller side? It seems like it might be for the remote node

Reply:
We need it in the ovirt_disk module. https://github.com/oVirt/ovirt-ansible-collection/blob/master/plugins/modules/ovirt_disk.py#L599